source/includes/_rsa-encryption.md.erb

```diff
@@ -9,11 +9,16 @@
 }
 ```
 
-Optionally, 2048-bit Asymmetric key encryption may be used when making a `POST` request to the API.
+Optionally, message encryption may be used when making a `POST` request to the API.
 
-- Request a public key from Q2, which will follow the PKCS #8 encryption scheme. You'll also be handed a key identifier.
+- Message encryption uses a combination of AES and RSA encryption.
+  - The message payload is encrypted using AES-CTR-256.
+  - The resultant AES key is then encrypted using an RSA 2048-bit public key.
+- An RSA key-pair can be created in the `Settings` area of the BDCS Nexus application and the public key can be copied.
```

Comment:
Should it be just "Nexus" instead of "BDCS Nexus"?

Reply:
@aaron-hynum-q2  Sure why not.